src/bootsampling.jl

```diff
@@ -103,6 +103,24 @@ end
 
 ExactSampling() = ExactSampling(0)
 
+"""
+[Maximum Entropy Sampling](https://cran.r-project.org/web/packages/meboot/vignettes/meboot.pdf)
+
+```julia
+MaximumEntropySampling()
+MaximumEntropySampling(100)
+maximumEntropySampling(100, MaximumEntropyCache())
+```
+
+NOTE: Implementation based off [pymeboot](https://github.com/kirajcg/pymeboot) as the original
```

Comment:
Since 'pymeboot' is MIT licensed, we should add "Copyright (c) 2017 Kira Huselius Gylling" to the license file of the package. And you may want to add your name, too.